manifests/server/config.pp

```diff
@@ -139,7 +139,6 @@
     concat { $pg_ident_conf_path:
       owner  => $user,
       group  => $group,
-      force  => true, # do not crash if there is no pg_ident_rules
```

Comment:
Concat module exactly raises a warning. This commit is about getting rid of the warning, as it can be confusing to users, as warnings are in red.